crates/prql-compiler/src/tests/test_error_messages.rs

```diff
@@ -189,6 +189,26 @@ fn test_basic_type_checking() {
     "###);
 }
 
+// See also test_bad_error_messages::misplaced_type_error
```

Comment:
```suggestion
// See also test_bad_error_messages::misplaced_type_error
// Note that the ``` Help: Type `bool` expands to `bool` ``` is not that useful
```